specs/BotBuilderOpenSourcePlans.md

```diff
@@ -0,0 +1,85 @@
+
+# External Speaking Engagments
+Requests for presentations at local groups should be submitted via GitHub Issues. Each request will be seperatly considered, and community members may choose to engage. As the community grows we hope this becomes an active part of our broad community.
+
+# Repo Layout and Usage
+
+In Github, we are going with 5 repos. These break down as:
+1. BotBuilder. This is the "root" repo were overviews, docs, and links to other repos are found. By design, this is the "landing page" for Bot developers. There should be no source-code found in this repo. 
+2. Botbuilder-dotnet. This is the C# Repo. Contained here is the V3 and V4 C# Code, along with tests, relevant samples, and issues. Links to the daily preview build, the public build, and other platform relevant materials are found here.
+3. Botbuilder-js.  This is the JavaScript Repo. Contained here is the V3 and V4 Javascript Code, along with tests, relevant samples, and issues. Links to the daily preview build for Node, the public build for Node, and other platform relevant materials are found here. 
+4. BotBuilder-Java. This is the Java Repo. Contained here is the V4 Java Code, along with tests, relevant samples, and issues. Links to the daily preview build, the public build, and other platform relevant materials are found here.
+5. BotBuilder-Python. This is the Python Repo. Contained here is the V4 Python Code, along with tests, relevant samples, and issues. Links to the daily preview build, the public build, and other platform relevant materials are found here.
+
+By design, most Issues and discussions should happen on the platform-agnostic "Botbuilder" repo. Issues that are incorrectly filed on a platform specific repo will be closed and re-opened on the correct repo. Likewise, platform specific issues will be moved as needed. 
+
+# Issue Tags
+The following tags are used across Repos:
+
+Tag | Description | Color
+--- | ----------- | -----
+Bug | The issue represents a bug. | Red
+Approved | The issue has been reviewed by the team and accepted as an issue to be resolved | Green
+Backlog | The issue has been approved by the team and added to the backlog work queue. | Yellow
+on-hold | The issue is acknowledged, but not activly being worked | Orange
+Docs | The issue in some way pertains to docs. Common uses are to report docs that are  incorrect, need enhancements, or generally unclear | Green
+Samples | The issue is pertains to samples. Common uses are to request new samples, changes to existing samples, or questions about a samples | Coral
+DCR | A Discussion thred about a potental change or new feature. | Purple
+Dialogs | Issue pertains to the dialog and prompt system | Pink
+Proactive | Issue relevant to proactive messaging | Dark Blue
+State-Management | Issue relevant to state management in the Bot | Light Yellow
+
```

Comment:
For generic feature area, I suggest the following format feature:State-Management. A bit long but easy to search and manage.